nomos-cli/src/cmds/disseminate/mod.rs

```diff
@@ -185,7 +196,31 @@ impl ServiceCore for DisseminateService {
                         );
                         std::process::exit(1);
                     }
-                    _ => {}
+                    Ok(Ok(cert)) => {
+                        if let Some(output) = settings.output {
+                            tracing::info!("Writing certificate to file...");
+                            std::fs::write(output, wire::serialize(&cert).unwrap())?;
+                            terminal_cmd_done();
+                        }
+
+                        if let Some(node_addr) = node_addr {
+                            let client = Client::new();
+                            tracing::info!("Sending certificate to node...");
+                            let res = client
+                                .post(node_addr.join(NODE_CERT_PATH).unwrap())
+                                .body(wire::serialize(&cert).unwrap())
+                                .send()
+                                .await?;
+                            if !res.status().is_success() {
+                                tracing::error!(
+                                    "Failed to send certificate to node: {}",
+                                    res.status()
+                                );
+                                std::process::exit(1);
+                            }
+                            terminal_cmd_done();
+                        }
```

Comment:
Nitpick: what if both options are activated? 🤔

Reply:
we do both?

Reply:
Yep my bad. Had the impresion that `terminal_cmd_done` finalized the process!!